client/src/components/ProfileForm.js

```diff
@@ -1,17 +1,43 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { Table, Form, Button, Checkbox, Icon, Popup } from 'semantic-ui-react';
+import { bindActionCreators } from 'redux';
+import { ProfileFormActions } from '../actions';
+import { Field, reduxForm } from 'redux-form';
+import { connect } from 'react-redux';
 import '../styles/ProfileForm.css';
 
-const ProfileForm = ({ userAttributes, preferences, handleCheckedBox }) => (
-	<Form className='container center scoreForm-form'>
-		<TableUserAttributes userAttributes={userAttributes}/>
-		<TablePreferences preferences={preferences} handleCheckedBox={handleCheckedBox}/>
-		<Button className='profle-reset'>Reset</Button>
-		<Button className='profile-submit'>Submit</Button>
-	</Form>
-)
+class ProfileForm extends Component {
+
+	submit = (values) => {
+		console.log(values)
+	}
+
+	render() {
+		console.log(this.props)
+		const { handleSubmit, pristine, reset, submitting, initialValues } = this.props
+		return (
+			<Form className='container center profileForm-form' onSubmit = {handleSubmit(this.submit)} >
```

Comment:
onSubmit = {handleSubmit(this.submit)} needs to have the two spaces before and after the "=" removed.
It should be: onSubmit={handleSubmit(this.submit)}